### ISSUE TITLE
Default brush size for label

### ISSUE THREAD

**Issue by Magistrus** [heartexlabs/label-studio#3675]
Is it possible to set brush size for each label in Labeling Interface?
![image](https://user-images.githubusercontent.com/1556270/218064110-439b50d3-6e78-4bcb-b42d-60c8b32f5203.png)

**Comment by makseq** [comment]
No, it's only one brush size for all labels. However, it's a good feature request.

**Comment by francoparodi** [comment]
Hi, this feature could be very useful... Do you know when it will be released? Thanks a lot.

**Comment by Walt9819** [comment]
Yes, this could be amazing!

**Comment by DungHD-1997** [comment]
i agree with this idea!

**Comment by zbyte** [comment]
That would be a very useful feature
